src/shims/x86/sse.rs

```diff
@@ -19,6 +19,10 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
         // Prefix should have already been checked.
         let unprefixed_name = link_name.as_str().strip_prefix("llvm.x86.sse.").unwrap();
         // All these intrinsics operate on 128-bit (f32x4) SIMD vectors unless stated otherwise.
+        // Many intrinsic names are sufixed with "ps" or "ss". "ps" means thet the operation is
+        // performed on each element of the vector, while "ss" means that the operation is
+        // performed only on the first element, copying the remaining elements from the input
+        // vector (for binary operations, from the left-hand side).
```

Comment:
Is there a mnemonic for this, like "ss = scalar \<something with s>" and "ps = \<something p> \<something s>"?

Reply:
I guess "ss = scalar single" and "ps = packed single" ("single" as in single precision FP) or something like that.

Reply:
That makes sense, can you add it to the comment?

Reply:
Done